src/assets/i18n/fr_FR.json

```diff
@@ -275,17 +275,19 @@
   "account.overview.wishlist.view_all": "Afficher toutes les listes de souhaits",
   "account.password.error.regexp": "Votre mot de passe doit comporter au moins 7 caractères.",
   "account.password.label": "Mot de passe actuel",
-  "account.payment.further_payments.heading": "Autres moyens de paiement sauvegardés",
+  "account.payment.further_payments.heading": "Autres modes de paiement",
   "account.payment.heading": "Informations de paiement enregistrées",
   "account.payment.link": "Paiement",
-  "account.payment.message": "Choisissez un moyen de paiement par défaut pour un paiement plus rapide de vos achats futurs.",
-  "account.payment.no_entries": "Actuellement, vous n’avez aucun moyen de paiement sauvegardé.",
-  "account.payment.payment_created.message": "Votre moyen de paiement par défaut a été enregistré sur votre compte.",
+  "account.payment.message": "Choisissez un mode de paiement par défaut pour un paiement plus rapide de vos achats futurs.",
+  "account.payment.no_entries": "Actuellement, vous n’avez aucun mode de paiement.",
+  "account.payment.no_preferred.message": "Vous n'avez plus sélectionné de mode de paiement préféré.",
+  "account.payment.no_preferred_method": "Pas de mode de paiement par défaut",
+  "account.payment.payment_created.message": "Votre mode de paiement par défaut a été enregistré sur votre compte.",
   "account.payment.payment_deleted.message": "Votre moyen de paiement a été supprimé.",
-  "account.payment.preferred.link": "Enregistrer comme moyen de paiement par défaut",
-  "account.payment.preferred_method": "Moyen de paiement par défaut",
+  "account.payment.preferred_method": "Votre mode de paiement par défaut",
   "account.payment.sepa_accepted_on": "Accepté le",
   "account.payment.sepa_mandate_Reference": "Référence du mandat",
+  "account.payment.standard_methods": "Méthodes de paiement standard",
```

Comment:
Please have Nicolas review the whole FR part. I have the feeling that this must be "Méthodes de paiement standards" but he will be sure about the correct grammar. ;-)

Reply:
"modes de paiement" is the correct wording